operations/rates.md

```diff
@@ -104,6 +104,116 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
+| `PricingType` | [Pricing type](rates.md#pricing-type) | required | Discriminator in which field inside `Pricing` contains additional data. |
+| `Pricing` | [Pricing](rates.md#rate-pricing-parameters) | required | Contains additional data about pricing of the rate. |
+
+### Rate pricing parameters
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `BaseRatePricing` | [Base rate pricing](rates.md#base-rate-pricing-parameters) | optional | Additional data for rate with base rate pricing. |
+| `DependentRatePricing` | [Base rate pricing](rates.md#dependent-rate-pricing-parameters) | optional | Additional data for rate with dependent rate pricing. |
+
+### Base rate pricing parameters
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Amount` | [Amount parameters](orders.md#amount-parameters) | required | Unit amount of the product. Note this overrides any previously defined amount. |
+| `NegativeOccupancyAdjustment` | decimal | required | This is the amount added to the price when occupancy of the space is less than the Space Category _Capacity_. To provide a discount price for under-occupancy, simply use a negative value. |
+| `ExtraOccupancyAdjustment` | decimal | required | This is the amount added to the price when the Space Category _Capacity_ is exceeded. |
+
+### Dependent rate pricing parameters
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `BaseRateId` | string | required | Unique identifier of the rate on which this dependent rate is based. |
+| `RelativeAdjustment` | decimal | required | This value is multiplied by the base rate to calculate an adjustment added to the base rate, e.g. a value of `1.0` means add 100% of the base rate to the base rate, effectively doubling the price. To subtract an amount, simply use a negative value, e.g. a value of `-0.1` means subtract 10% of the base rate from the base rate. |
+| `AbsoluteAdjustment` | decimal | required | This value is simply added to the base rate to obtain the new price. To subtract an amount, use a negative value. |
+
+### Response
+
+```javascript
+{
+    "Rates": [
+        {
+            "BaseRateId": null,
+            "BusinessSegmentId": null,
+            "GroupId": "b9f25a45-9b9a-4b33-99bd-b06f008eb6f5",
+            "Id": "ed4b660b-19d0-434b-9360-a4de2ea42eda",
+            "ServiceId": "bd26d8db-86da-4f96-9efc-e5a4654a4a94",
+            "IsActive": true,
+            "IsEnabled": true,
+            "IsPublic": true,
+            "Type": "Public",
+            "Name": "My rate",
+            "ShortName": "FF",
+            "ExternalNames": {
+                "en-US": "Long Stay Flexible Rate"
+            },
+            "ExternalIdentifier": "D001"
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Rates` | array of [Rate](#rate) | required | Rates that have been added. |
+
```

Comment:
`BaseRateId` is shown as `required` but is "null" in the JSON example. I assume this would be null for an actual base rate, so shouldn't it be `optional`?